src/components/manage/Widgets/ArrayWidget.jsx

```diff
@@ -325,7 +325,8 @@ class ArrayWidget extends Component {
               : this.props.choices
               ? [
                   ...choices,
-                  ...(this.props.noValueOption && !this.props.default
+                  ...(this.props.noValueOption &&
+                  typeof this.props.default === undefined
```

Comment:
```suggestion
                  this.props.default === undefined
```

`typeof undefined` returns the string `"undefined"`, which does not equal the object `undefined`

Reply:
@davisagli I fixed it. Thanks!